portfolio/src/main/java/com/google/sps/servlets/UserInfoServlet.java

```diff
@@ -69,16 +70,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String nickname = request.getParameter("nickname");
     String email = userService.getCurrentUser().getEmail(); // user email is used as id
 
-    //check that input is non-empty
+    // check that input is non-empty
     if (nickname.isEmpty()) {
-        response.sendRedirect("/");
-        return;
+      response.sendRedirect("/#!/profile-page");
+      return;
     }
 
     User newUser = User.create(email, nickname);
     userStorage.upload(newUser);
 
-    response.sendRedirect("/");
+    response.sendRedirect("/#!/profile-page");
```

Comment:
nit: can we make "/#!/profile-page" into a constant string literal to ensure consistency?